demos/textfield.html

```diff
@@ -195,6 +205,61 @@
           }, 0);
         </script>
       </section>
+
+      <section class="example">
+        <h2>Textfield - Leading/Trailing icons</h2>
+        <div class="demo-note">
+          <em>
+            Note: Some implementations utilizing leading and trailing icons may wish to set a width on the
+            <code>input</code> element to acheive uniform widths on textfields.
+            This demo acheives that by adding a custom class name: <code>custom-textfield-input</code> and setting
+            some style:
+          </em>
+          <br />
+          <pre>
```

Comment:
Looks correct in the demo. The leading spaces thing is why the code itself is aligned to the start of each line.